yw_website/yw_website/apps/website/models.py

```diff
@@ -43,8 +43,19 @@ class Version(models.Model):
     last_modified = models.DateTimeField()
 
 
+class Script(models.Model):
+    script_check_sum = models.CharField(max_length=128)
+    script = models.FileField(
+        upload_to='workflow_scripts/', null=True, default=None)
+
+
+class VersionScript(models.Model):
```

Comment:
not a many to many relationship unless tim said otherwise

Reply:
fixed